### ISSUE TITLE
Webform admin dialogs unreadable/inaccessible text

### ISSUE THREAD

**Issue by aslaymoore** [az-digital/az_quickstart#5949]
## Problem/Motivation
Webform admin dialogs have unreadable/inaccessible text due to color contrast of blue on black/dark gray, and grey text over a gray highlight background. `az-core` provides [CSS for `.ui-dialog`](https://github.com/az-digital/az_quickstart/blob/095171a69f2d700a3b3d461deb3ee6134be85442/modules/custom/az_core/css/arizona-bootstrap-node-form-overrides.css#L58), but those styles are not making it to Webforms in the admin interface. Headings are defaulting to blue for h1-h6.

This seems like it's only the case on modal dialogs, and not on the sidebar dialog when editing form elements.

<img width="701" height="535" alt="Image" src="https://github.com/user-attachments/assets/5dd2e54b-0f59-4ea1-b1fd-8ddf9c3e0d0c" />

https://github.com/user-attachments/assets/e304a4cf-731b-49f4-be4e-31ac1ce82ec1

### To Reproduce
1. Create a Webform (or edit an existing one)
2. Perform an action that triggers a modal dialog to appear (such as trying to create or delete a Webform)

## Proposed resolution
Ensure `az-core` styles for `ui-dialog` reach Webforms too. Or add CSS that accounts for Webform dialogs.

### Expected behavior
Text in Webform admin dialogs should be readable and have accessible color contrast.
